src/intervals/conversion_promotion.jl

```diff
@@ -2,14 +2,12 @@
 
 ## Conversion and promotion
 
-## Default conversion is to Interval{Float64}
+## Default conversion to Interval, corresponds to Interval{Float64}
 convert{T<:Real}(::Type{Interval}, x::T) = make_interval(Float64, x)
-@compat convert{T<:Union{Float64,BigFloat,Rational}}(::Type{Interval}, x::T) =
-    make_interval(T, x)
 
-## Conversion to intervals (with rounding) from Integer or Irrational
+## Conversion to specific type intervals
 @compat convert{T<:Union{Float64,BigFloat}, S<:Real}(::Type{Interval{T}}, x::S) =
```

Comment:
Maybe we should define `RestrictedFloat` to be `Union{Float64,BigFloat}`; that may make the code more readable.

As for `Rational`, the nice thing is precisely that there is no rounding!, so if you have to deal e.g. with polynomials with rational coefficients, the results do not need to be rounded! However, as soon as you include `sqrt` or things more involved, `Rational` intervals are not useful any more.

This is only to say that I have mixed feelings on this... I propose to keep it there for the time being.